src/ucp/proto/proto_common.inl

```diff
@@ -362,6 +362,14 @@ ucp_proto_request_pack_rkey(ucp_request_t *req, ucp_md_map_t md_map,
                   md_map);
     }
 
+    if ((memh->flags & UCP_MEMH_FLAG_HAS_GVA) &&
```

Comment:
1. maybe do this before line 360: so if we disable gva, we still allow partial registration or registration failure, and don't fail the rkey packing completely.
2. Pls add comment before this code block to explain why we are disabling GVA here
3. Maybe extend GVA_ENABLE param to yes/no/auto - when auto means that we disable it with error handling. yes means always enable, even if may cause data corruption in error flow. default is still no, though.